public/static/app/perf-bb/ExpandedMetricsSignalProcessingController.js

```diff
@@ -36,7 +36,7 @@ function handleResponse(result, $scope) {
         measurement: cp.measurement,
         percent_change: cp.percent_change.toFixed(2),
         triage_status: cp.triage.triage_status,
-        thread_level: cp.thread_level,
+        thread_level: cp.arguments["thread_level"],
```

Comment:
Do we need to worry about the case where a change point doesn't have a thread_level? Will that null case affect anything in the table downstream from this?

Reply:
Thread level *should* just not be displayed, since the dict it will return `undefined`.

Reply:
Sounds good